test/utils.ts

```diff
@@ -76,7 +85,16 @@ export function randomAddress() {
 }
 
 export async function getParamType(contractName: string, functionName: string, paramName: string) {
-  const contractFactory = await getContractFactory(contractName, new ethers.VoidSigner(zeroAddress));
+  const contractFactory = await getContractFactory(contractName, new ethers.VoidSigner(ethers.constants.AddressZero));
   const fragment = contractFactory.interface.fragments.find((fragment) => fragment.name === functionName);
   return fragment!.inputs.find((input) => input.name === paramName) || "";
 }
+
+export async function createMock(contractName: string) {
```

Comment:
nit: rename to `createFake` since mocks are separate from fakes! For example:

>One powerful feature of a mock is that you can modify the value of variables within the smart contract. You could, for example, use this feature to test the behavior of a function that changes behavior depending on the value of a variable.